csrankings-b.csv

```diff
@@ -436,6 +436,7 @@ Bhubaneswar Mishra,New York University,https://cs.nyu.edu/mishra,kXVBr20AAAAJ
 Bhujanga Chakrabarti,Victoria University of Wellington,http://sms.victoria.ac.nz/Main/BhujangaChakrabarti,3VT_IZEAAAAJ
 Bhuvan Urgaonkar,Pennsylvania State University,http://www.cse.psu.edu/~buu1,ZoI6hPwAAAAJ
 Bhuwan Dhingra,Duke University,http://www.cs.cmu.edu/~bdhingra/,2W2ttrQAAAAJ
+Bhuvana Krishnaswamy,University of Wisconsin - Madison,https://uwconnect.ece.wisc.edu/,1Hn1s1sAAAAJ

```

Comment:
ECE, not CS; please clarify how they meet the inclusion criteria.

Reply:
Bhuvana is also recently affiliated with the CS department at UW-Madison. Please find her name on the CS faculty website: https://www.cs.wisc.edu/people/faculty/